code/modules/antagonists/cult/cult_items.dm

```diff
@@ -833,6 +833,8 @@
 	var/C = user.client
 	if(ishuman(user) && C)
 		angle = mouse_angle_from_client(C)
```

Comment:
does this line need to be here anymore?

Reply:
I thought I deleted it, let me check

Reply:
I know I can get rid of angle very easly, not sure about the first 2 lines.